homeassistant/components/prometheus/__init__.py

```diff
@@ -345,6 +361,11 @@ def _labels(state: State) -> dict[str, Any]:
             "entity": state.entity_id,
             "domain": state.domain,
             "friendly_name": state.attributes.get(ATTR_FRIENDLY_NAME),
+            "area": state.attributes.get(ATTR_AREA_ID),
+            # TODO: how to get zone here?
+            # "zone": state.attributes.get(ATTR_AREA_ID),
+            "entity_id": state.object_id,
```

Comment:
If this is called `object_id` internally, could you please use `object_id` for the label name as well?